definitions/divorce/json/CaseEvent.json

```diff
@@ -2742,8 +2742,7 @@
     "Name": "Bulk Scanning - Case Create",
     "Description": "Create a case from scanned documents",
     "DisplayOrder": 20,
-    "PreConditionState(s)": "*",
-    "PostConditionState": "*",
```

Comment:
what about PreConditionState?

Reply:
We can't have one. This is the event that starts it all.